pkg/backup/http.go

```diff
@@ -34,6 +35,7 @@ const (
 func (b *Backup) startHTTP() {
 	http.HandleFunc(backupapi.APIV1+"/backup", b.serveSnap)
 	http.HandleFunc(backupapi.APIV1+"/backupnow", b.serveBackupNow)
+	http.HandleFunc(backupapi.APIV1+"/status", b.serveStatus)
```

Comment:
We also need to update backup_service.md . Are you going to do it in separate PR?